packages/projection/test/operators/Mappers/withHandles.test.ts

```diff
@@ -288,4 +290,40 @@ describe('withHandles', () => {
       expect(handles[0].latestOwnerAddress).toBe(maryAddress);
     });
   });
+
+  describe('subhandles', () => {
```

Comment:
Missing test coverage for virtual handle owner address that comes from datum metadata

Reply:
addressed: 998e07df7ceb955a5f76928d37050d083073d390